include/exec/static_thread_pool.hpp

```diff
@@ -112,10 +112,66 @@ namespace exec {
           >;
 
       template <class SenderId, class ReceiverId, class Shape, class Fun, bool MayThrow>
-        struct bulk_shared_state : task_base {
+        struct bulk_shared_state {
           using Sender = stdexec::__t<SenderId>;
           using Receiver = stdexec::__t<ReceiverId>;
 
+          struct bulk_task : task_base {
+              bulk_shared_state *sh_state_;
```

Comment:
prefer two-space indents. I can has .clang-format now? :-)

Reply:
Not yet :) I've reformatted the code, thank you for noticing that!